dev/com.ibm.ws.security.javaeesec/resources/com/ibm/ws/security/javaeesec/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -19,45 +19,41 @@
+JAVAEESEC_ERROR_NO_VALIDATION.explanation=No IdentityStore object was found to validate the user. Therefore, the user cannot be authenticated.
+JAVAEESEC_ERROR_NO_VALIDATION.useraction=Make sure that at least one IdentityStore object supports validation of the user.
 
-JAVAEESEC_ERROR_NO_HAM=CWWKS1912E: The HttpAuthenticationMechanism for the {0} module in the {1} application could not be created.
-JAVAEESEC_ERROR_NO_HAM.explanation=The HttpAuthenticationMechanism cannot be created.
-JAVAEESEC_ERROR_NO_HAM.useraction=Investigate any error messages from the HttpAuthenticationMechanism.
+JAVAEESEC_ERROR_NO_HAM=CWWKS1912E: The HttpAuthenticationMechanism object for the {0} module in the {1} application could not be created.
+JAVAEESEC_ERROR_NO_HAM.explanation=The reason that the HttpAuthenticationMechanism object cannot be created varies. The error messages for the HttpAuthenticationMechanism object provide information on why the the HttpAuthenticationMechanism object cannot be created.
+JAVAEESEC_ERROR_NO_HAM.useraction=Investigate any error messages from the HttpAuthenticationMechanism object and make corrections based on the error messages.
 
 #Note to translator, do not translate EL.
-JAVAEESEC_WARNING_IDSTORE_CONFIG=CWWKS1916W: There was an error evaluating the Expression Language (EL) expression for the ''{0}'' attribute of the identity store annotation. Ensure that the EL expression and the result are valid and ensure any referenced beans that are used in the expression are resolvable. The default attribute value ''{1}'' will be used.
-JAVAEESEC_WARNING_IDSTORE_CONFIG.explanation=The identity store attribute Expression Language expression cannot be resolved to a valid value.
-JAVAEESEC_WARNING_IDSTORE_CONFIG.useraction=Make sure that the annotation contains a valid configuration value. If you use EL expressions, ensure that they are valid and that any referenced beans that are used in the expression are resolvable.
+JAVAEESEC_WARNING_IDSTORE_CONFIG=CWWKS1916W: The Expression Language (EL) expression for the ''{0}'' attribute of the identity store annotation cannot be resolved to a valid value. Ensure that the EL expression and the result are valid and ensure that any referenced beans that are used in the expression are resolvable. The default attribute value of ''{1}'' is used instead.
+JAVAEESEC_WARNING_IDSTORE_CONFIG.explanation=The cause of the error is mismatch of the type between the EL result and the expected attribute value. For example, if the expected attribute type is String, the EL result needs to be String.
+JAVAEESEC_WARNING_IDSTORE_CONFIG.useraction=Make sure that the annotation contains a valid configuration value. Ensure that the EL expressions are valid and that any referenced beans that are used in the expression are resolvable and the type of the result corresponds with the attribute.
```

Comment:
Change "Ensure that the EL expressions are valid and that any referenced beans that are used in the expression are resolvable and the type of the result corresponds with the attribute." 
to
Ensure that the EL expressions are valid, that any referenced beans that are used in the expression are resolvable, and that the type of the result corresponds with the attribute.

Reply:
done.